document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -236,7 +238,10 @@ protected String parseQueryForNonCompositeFilter(Filter filter) {
         List<Object> values = (List<Object>) value;
         String collect = values
             .stream()
-            .map(val -> "'" + val + "'")
+            .map(val -> {
+              paramsBuilder.addStringParam(String.valueOf(val));
```

Comment:
Where are we adding int/long/double params?

Reply:
Only for the cases where we might be querying `id` directly